docs/cluster-configuration.md

```diff
@@ -892,7 +892,27 @@ metadata:
         }
       ]
 ```
-
+##### Disable DataVolume garbage collection
+To disable DataVolume garbage collection the following annotation should be added to the HyperConverged CR:
```

Comment:
Please add a (very) short background, or a link to relevant documentation.

Also - nit:
```suggestion
To disable DataVolume garbage collection, the following annotation should be added to the HyperConverged CR:
```